### ISSUE TITLE
[new site support] opac.lib.takushoku-u.ac.jp

### ISSUE THREAD

**Issue by KRAZIKATZ** [lovasoa/dezoomify#740]
### Site name and desciption
[opac.lib.takushoku-u.ac.jp](https://opac.lib.takushoku-u.ac.jp/kyugaichi/htmls/index.html)

Website with archived and collected materials from Takushoku University JP, regarding historical material such as documents, maps, etc about former Imperial Japanese colonial holdings before the end of World War II. Has a range of unique high resolution images that are very rare, unique, and important for those interested in historical research of this topic. 

### Example URLs
[https://opac.lib.takushoku-u.ac.jp/kyugaichi/htmls/pages/2011_08.html](https://opac.lib.takushoku-u.ac.jp/kyugaichi/htmls/pages/2011_08.html)
_Wasn't able to find metadata url_
<img width="993" alt="Screenshot 2023-07-05 at 00 00 46" src="https://github.com/lovasoa/dezoomify/assets/138654252/8f0e9d4f-1001-4c7e-9705-498ace0f8423">

### Current error message
```
Uncaught Error: Unable to find a proper dezoomer for:
https://opac.lib.takushoku-u.ac.jp/kyugaichi/htmls/pages/2011_08.html

(https://dezoomify.ophir.dev/dezoomers/automatic.js:30)`
```

**Comment by DigitalCurator** [comment]
[Resolved]

**Comment by KRAZIKATZ** [comment]
Thank you so much!

**Comment by mappingmonkey** [comment]
@DigitalCurator Could provide some hints on how resolved the URL please

**Comment by Arekkusu1998** [comment]
I agree with @mappingmonkey: thanks for your kind help, @DigitalCurator, but we'd be interested to know what to write, both on dezoomify and possibly on [dezoomify-rs](https://github.com/lovasoa/dezoomify-rs). It would be useful for everyone.

**Comment by AmiralCrapaud** [comment]
Actually perhaps that might be the same issue I have with my recent query #776 considering these Japanese archive websites seem to share a lot of things when it comes to architecture & interface...
So  @DigitalCurator if you're still around, I'd be happy to know more, for certain ^^ big thanks :)

**Comment by Yovahti** [comment]
<img width="1899" height="873" alt="Image" src="https://github.com/user-attachments/assets/4755af4b-6436-42cb-bca0-96b6dbfaa76c" />
I cant use dezoomify to download map on this website now, how can you solve the problem?

**Comment by AlexanderDeLargeBrotherSir** [comment]
<img width="3000" height="2329" alt="Image" src="https://github.com/user-attachments/assets/66c0571d-cb61-42ae-ba28-994142cf5fcb" />

dezoomify-rs did not work here so i made a python script instead

Steps to download

1) Install python (if haven't already) - winget install Python.Python (copy & paste in cmd)
2) py -m pip install requests pillow - copy & paste in cmd
3) [takushoku_downloader.py](https://github.com/user-attachments/files/31321141/takushoku_downloader.py) - python takushoku_downloader.py (copy & paste in cmd)
(IMP - put the script inside a folder and run cmd from it for this to work)
4) Sit back it will download tiles and stitch them in png and tiff

@KRAZIKATZ  
@AmiralCrapaud 
@mappingmonkey 
@Arekkusu1998 

You all will express your Profoundest of Gratitude towards Sir Alexander DeLarge and you will close this issue at once.
